tl.tl

```diff
@@ -1275,6 +1280,7 @@ parse_statements = function(tokens: {Token}, i: number, errs: {ParseError}): num
 end
 
 function tl.parse_program(tokens: {Token}, errs: {ParseError}): number, Node
+   errs = errs or {}
```

Comment:
This here is so tl would not crash if there are parse errors and you did not supply an `errs` parameter (tests don't do that). From what i understand the `errs` parameter is supposed to be optional?

Reply:
Yeah, since this is the entry point of the parser it makes sense to keep this optional. :+1: